iOS/swiftui-davinci/Davinci/Davinci/ContentView.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ContentView.swift
+//  Davinci
+//
+//  Copyright (c) 2024 Ping Identity. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+  @State private var startDavinici = false
+  @State private var path: [String] = []
+  
+  var body: some View {
+    
+    NavigationStack(path: $path) {
+      List {
+        NavigationLink(value: "Davinci") {
+          Text("Launch Davinci")
+        }
+        NavigationLink(value: "Token") {
+          Text("Access Token")
+        }
+        NavigationLink(value: "User") {
+          Text("User Info")
+        }
+        NavigationLink(value: "Logout") {
+          Text("Logout")
+        }
+      }.navigationDestination(for: String.self) { item in
+        switch item {
+        case "Davinci":
+          DavinciView(path: $path)
```

Comment:
`DavinciView` is missing. Some others in this switch statement were removed intentionally, so should be removed here.